DEEP_DIVES/46-better-file-chunking.md

```diff
@@ -1,15 +1,75 @@
 # Better File Chunking
 
-<Short summary of topic including Open Problem(s) to solve>
+Within the IPFS stack/ecosystem, just as within computing as a whole, **an
+uncompressed stream of untagged octets** is a fundamental unit of exchange.
+As a general-purpose data storage system IPFS needs to handle an unbounded
+variety of content represented by such streams. Handling the maximum amount
+of this variety efficiently ( ideally **by default** ) would likely have an
+outsized impact on the future adoption of IPFS as a **long-term data interchange
+medium/format.**
+
+The current stream-chunking options provided by the "official" IPFS
+content-adders are not particularly good. If left unchecked, the
+"evolutionaly pressure" will inevitably lead to proliferation of chunking
+algorithms within and beyond ProtocolLabs-controlled projects. This could
+be undesireable as any iteration would result in radically different
+IPFS addresses, hampering user experience, e.g.:
+
+* Preventing users from recognizing identical content via simple "eyeballing a list of hashes"
+
+* Increaed storage requirements due to failing de-duplication of identical content
+
+* Increased retrieval times due to high counts of not-already-present blocks
+
+This session is mostly a "conversation starter" to properly map the available
+problem ( and solution ) spaces. I would strongly prefer to keep the deep dive
+light on specific technical details and instead **focus on wider architectural /
+UX / game-theory-ish problems.**
 
 ## Goal
 
-<Define what we would get if the Open Problem(s) are solved for this Deep Dive>
+It would be great if towards the end of this session we have answers or maybe
+even consensus 🤞 regarding the following deceptively simple
+checklist:
+
+- [ ] Identify a ~dozen-or-two of realistic user-stories where IPFS could
```

Comment:
"~dozen-or-two" - what does this mean?

Reply:
@alanshaw I got carried away without thinking D:
Let's add https://github.com/ipfs/camp/pull/59 on top